src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/DataElementStorageTests.cs

```diff
@@ -65,6 +68,7 @@ public void Dispose()
         {
             string requestUri = $"{versionPrefix}/instances?org={testOrg}";
 
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _validToken);
```

Comment:
In my opinion, it is more readable to put this in the constructor instead of having this line in each method.
Will that cover the needs of the tests?